app/controllers/idv/otp_delivery_method_controller.rb

```diff
@@ -67,7 +67,13 @@ def render_new_with_error_message
     def send_phone_confirmation_otp_and_handle_result
       save_delivery_preference
       result = send_phone_confirmation_otp
-      analytics.idv_phone_confirmation_otp_sent(**result.to_h)
+      analytics.idv_phone_confirmation_otp_sent(
+        **result.to_h.merge(
+          {
+            adapter: Telephony.config.adapter,
+          },
+        ),
```

Comment:
style, i'd drop the extra curly braces and one-line this:
```suggestion
        **result.to_h.merge(adapter: Telephony.config.adapter),
```

Reply:
Thank you, was in a battle with Rubocop to make it look reasonable (I didn't win)

Reply:
oh I also just realized.... `**` can merge to!
```suggestion
        adapter: Telephony.config.adapter,
        **result.to_h,
```